packages/collapsible/src/LionCollapsible.js

```diff
@@ -76,6 +72,26 @@ export class LionCollapsible extends LitElement {
     }
   }
 
+  toggle() {
+    this.__toggle();
+  }
+
+  expand() {
+    if (!this.expanded) {
+      this.expanded = true;
+      this._updateContentSize(this.expanded);
+      this.__dispatchEvent(this.expanded);
+    }
+  }
+
+  collapse() {
```

Comment:
Could we rename this to `hide`, to stay inline with our other components? (e.g. overlays)

Reply:
👍